creator/creator_test.go

```diff
@@ -2802,7 +2802,7 @@ ET
 		t.Errorf("Fail: %v\n", err)
 		return
 	}
-	if fileInfoOptimized.Size() >= fileInfo.Size() {
+	if fileInfoOptimized.Size() > fileInfo.Size() {
```

Comment:
Any idea why this changed?  Some change in the content streams due to this change?

Reply:
The test was adding a raw stream directly to the page. Because the components were rendered on the page as soon as the Draw method was called (which flate encodes the content), and the raw stream was added afterwards, the optimizer found something to compress. But now, the compression of the page stream is done in the Write method. So, everything is merged and compressed there. So there is no difference between the files in size as there is nothing to compress anymore at that point.